### PR TITLE
cleaning the arduino file

### PR DESCRIPTION
**Pull request by thebascos**
## Description

_Provide a description on what was changed/added/fixed in this PR_

## Checklist

_First, create a draft pull request. Then mark the PR as ready for review when all necessary checkbox items has been completed_

- [ ] Reviewed own code
- [ ] Commented on code that is hard to understand
- [ ] Implemented tests for the feature/bugfix
- [ ] All GitHub status checks pass
- [ ] The frontend and backend PR previews have been deployed

**Backend only**

- [ ] Generated the client api if there are new or deleted endpoints and/or DTOs

### CODE REVIEW

File: logistechMOTHAFUCKA/logistechMOTHAFUCKA.ino [hunk 1]
```diff
@@ -7,33 +7,27 @@
...
 
 String serverName = "http://192.168.1.9:5001/api/v1/gps/product-item";
-unsigned long lastTime = 0;
-unsigned long timerDelay = 5000;
 
 //static const int RXPin = 5, TXPin = 6;
 SoftwareSerial ss (5, 6); //same as wiegand???
 
+
+uint32_t codes[MAX_CODES];   // Array to store previous codes
```
Comment (thebascos) [inline logistechMOTHAFUCKA/logistechMOTHAFUCKA.ino:28]: @princhcanal make this dynamic

File: logistechMOTHAFUCKA/logistechMOTHAFUCKA.ino [hunk 2]
```diff
@@ -169,10 +113,34 @@ void loop() {
...
+      if (httpResponseCode>0) {
+        Serial.print("HTTP Response code: ");
+        Serial.println(httpResponseCode);
+        String payload = http.getString();
+        Serial.println(payload);
+      }
+      else {
+        Serial.print("Error code: ");
+        Serial.println(httpResponseCode);
+      }
```
Comment (thebascos) [inline logistechMOTHAFUCKA/logistechMOTHAFUCKA.ino:134]: @princhcanal  you may simplify this part too. like the validation of the response we can put it outside` if(is_unique)` and `else if(! if(is_unique))` so it wont repetitive

File: logistechMOTHAFUCKA/logistechMOTHAFUCKA.ino (hunk 1 above)
Comment (princhcanal) [reply logistechMOTHAFUCKA/logistechMOTHAFUCKA.ino:28]: duly noted 📝

File: logistechMOTHAFUCKA/logistechMOTHAFUCKA.ino (hunk 2 above)
Comment (princhcanal) [reply logistechMOTHAFUCKA/logistechMOTHAFUCKA.ino:134]: duly noted as well 📓

**Review by princhcanal — APPROVED** [review]
감사합니다 🙏🏻

**Comment by thebascos** [comment]
> 감사합니다

엿먹어😊

**Pull request merged**